test/built-ins/Promise/race/resolve-ignores-late-rejection-deferred.js

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2020 Rick Waldron, 2020 Rick Waldron. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: >
+  Resolved promises ignore rejections through deferred invocation of the
+    provided resolving function
+esid: sec-promise.race
+info: |
+  Let result be PerformPromiseRace(iteratorRecord, C, promiseCapability, promiseResolve).
+
+  PerformPromiseRace
+
+  Repeat
+    ...
+    Perform ? Invoke(nextPromise, "then", « resultCapability.[[Resolve]], resultCapability.[[Reject]] »).
+
+flags: [async]
+features: [arrow-function]
+---*/
+
+let resolver = {
+  then(resolve) {
+    new Promise((resolve) => resolve())
+      .then(() => resolve(42));
+  }
+};
+let lateRejector = {
+  then(resolve, reject) {
+    new Promise((resolve) => resolve())
+      .then(() => {
+        resolve(9);
+        reject();
+      });
+  }
+};
+
+Promise.race([resolver, lateRejector])
```

Comment:
Would this test be more relevant if it had only lateRejector, no resolver? That'd test that the rejection is properly ignored and the value is what we resolved with (before rejecting), in this case 9.

Now the value is coming from a separate Promise entirely.